bin/build

```diff
@@ -65,15 +64,33 @@ findFile() {
   head -n 1
 }
 
-# Recursively search input directory for a package.json
+# Recursively search output directory for a package.json
 PACKAGE_PATH=$(findFile "package.json" $OUTPUT_DIR/bundle/programs/server)
+export PACKAGE_PATH
 
-if [[ $PACKAGE_PATH ]]; then
-  export PACKAGE_PATH
+NODE_VERSION=$(get-version --engine node $PACKAGE_PATH 2>/dev/null) || true
+NPM_VERSION=$(get-version --engine npm $PACKAGE_PATH 2>/dev/null) || true
 
```

Comment:
note: `get-version` always returns a version, right? I don't think you need to check if these variables got set below